packages/tentales/src/service/create-service-callers.ts

```diff
@@ -1,13 +1,16 @@
 import fetch from "node-fetch"
 import { createLog } from "tentales-log"
 import { Action, ServiceConfig, ServiceCaller, Config } from "tentales"
+import { mintToken } from "../utils/token"
 
 // tslint:disable-next-line:no-any
 function getServiceConfig(serviceName: string, config: any): ServiceConfig {
   return config.services[serviceName] as ServiceConfig
 }
 
 export function createServiceCallers(config: Config): ServiceCaller[] {
+  const bearer = `Bearer ${mintToken(config.auth.serverSecret)}`
```

Comment:
Could we just call the prop `baerer` instead of `serverSecret`?

Reply:
But it's not really a bearer, is it? It's a secret which is used to mint a token, which is is then sent in the authorization header as the _type_ `bearer`

Reply:
Makes sense